rules.ini

```
@@ -59,6 +59,7 @@ Danmakufu = (?:^|/)th_dnh\.def$
 Defold = (?:^|/)game\.dmanifest$
 Diesel = ^dieselx\.cfg$
 EngineX = ^(?:_bin_PC/BuildData|data)/FILELIST.BIN$
+Essence_Engine = \.sga$
```

```suggestion
Essence_Engine = Data\.sga$
```
From what i saw it should fix all false positive although i'm not sure every of their games are matched, we'll have to see after it gets deployed